spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-functional-sample/pom.xml

```diff
@@ -9,14 +9,14 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>spring-cloud-gcp-pubsub-stream-binder-functional-sample</artifactId>
+  <artifactId>spring-cloud-gcp-pubsub-stream-functional-sample</artifactId>
   <packaging>pom</packaging>
-  <name>Spring Cloud GCP Code Sample - Pub/Sub Stream Binder (Functional style)</name>
+  <name>Spring Cloud GCP Code Sample - Pub/Sub Stream (Functional style)</name>
```

Comment:
Would go more verbose here, with either leaving Binder (in case people actually search for the binder) or specifying "Spring Cloud Stream".